packages/strapi-plugin-upload/admin/src/components/FiltersPicker/FiltersCard/FilterInput.js

```diff
@@ -9,7 +9,7 @@ import PropTypes from 'prop-types';
 import { DateTime } from '@buffetjs/custom';
 import { InputText, Select } from '@buffetjs/core';
 
-import SizeInput from '../SizeInput';
+import SizeInput from './SizeInput';
```

Comment:
Line 30:

If you want to disable the autoComplete you need to add ` autoComplete="new-password"`

Reply:
We already set a `autocomplete="off"` by default (InputNumber from buffetjs/core)